lms/djangoapps/teams/tests/test_api.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+"""
+Tests for Python APIs of the Teams app
+"""
+from __future__ import absolute_import
+
+from uuid import uuid4
+import unittest
+
+import ddt
+import pytz
+from mock import Mock
+from opaque_keys.edx.keys import CourseKey
+
+from lms.djangoapps.teams import api as teams_api
+from lms.djangoapps.teams.tests.factories import CourseTeamFactory
+from student.tests.factories import CourseEnrollmentFactory, UserFactory
+from xmodule.modulestore.tests.django_utils import SharedModuleStoreTestCase
+
+COURSE_KEY1 = CourseKey.from_string('edx/history/1')
+COURSE_KEY2 = CourseKey.from_string('edx/history/2')
+
+DISCUSSION_TOPIC_ID = uuid4().hex
+
+class APITests(SharedModuleStoreTestCase):
```

Comment:
Consider calling this `PythonAPITests`, and saying to API functions instead of endpoints, otherwise an unsuspecting dev may think these are REST API tests.